src/ClientInterface.php

```diff
@@ -108,7 +108,7 @@
  * @method Status            ftaliasdel(string $alias)
  * @method Status            ftaliasupdate(string $alias, string $index)
  * @method Status            ftalter(string $index, Schema $schema, ?AlterArguments $arguments = null)
- * @method Status            ftcreate(string $index, Schema $schema, ?CreateArguments $arguments = null)
+ * @method Status            ftcreate(string $index, FieldInterface[] $schema, string $on = '', array $prefixes = [], string $filter = '', string $language = '', string $languageField = '', float $score = 0, string $scoreField = '', string $payloadField = '', bool $maxTextFields = false, int $temporary = 0, bool $noOffsets = false, bool $noHl = false, bool $noFields = false, bool $noFreqs = false, array $stopWords = [], bool $skipInitialScan = false)
```

Comment:
Maybe temporary should become temporarySeconds to make it more user friendly - even though the arg name is _temporary_

Reply:
Okay 👍

Reply:
Fixed